### ISSUE TITLE
PR not created on prefix 'deps' using 'terraform-module' release-strategy

### ISSUE THREAD

**Issue by kstevensonnv** [google-github-actions/release-please-action#922]
### TL;DR

PR not created on prefix 'deps' using 'terraform-module' release-strategy.

### Expected behavior

PR should be created as a dependency update is a user-facing change.

The release-please documentation lists it as a "releasable-unit":

### Step 1: Ensure releasable units are merged

Release Please creates a release pull request after it notices the default branch
contains "releasable units" since the last release.
A releasable unit is a commit to the branch with one of the following
prefixes: "feat", "fix", **and "deps"**.
(A "chore" or "build" commit is not a releasable unit.)

### Observed behavior

PR not created:

>No user facing commits found since 61da1eb745fdff3bd6dd89eb4d89df4bd4c9fda1 - skipping

### Action YAML

```yaml
on:
  push:
    branches:
      - main

permissions:
  contents: write
  pull-requests: write

name: release-please

jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v4
        with:
          token: ${{ secrets.RELEASE_PLEASE_GITHUB_TOKEN }}
          release-type: terraform-module
```

### Log output

```text
2024-01-05T19:44:18.9912973Z Requested labels: ubuntu-latest
2024-01-05T19:44:18.9913303Z Job defined at: org/repo/.github/workflows/release-please.yml@refs/heads/main
2024-01-05T19:44:18.9913637Z Waiting for a runner to pick up this job...
2024-01-05T19:44:19.6099231Z Job is waiting for a hosted runner to come online.
2024-01-05T19:44:24.3026096Z Job is about to start running on the hosted runner: GitHub Actions 2 (hosted)
2024-01-05T19:44:27.1740352Z Current runner version: '2.311.0'
2024-01-05T19:44:27.1777431Z ##[group]Operating System
2024-01-05T19:44:27.1778882Z Ubuntu
2024-01-05T19:44:27.1779339Z 22.04.3
2024-01-05T19:44:27.1780115Z LTS
2024-01-05T19:44:27.1780817Z ##[endgroup]
2024-01-05T19:44:27.1781583Z ##[group]Runner Image
2024-01-05T19:44:27.1782700Z Image: ubuntu-22.04
2024-01-05T19:44:27.1783323Z Version: 20231217.2.0
2024-01-05T19:44:27.1785276Z Included Software: https://github.com/actions/runner-images/blob/ubuntu22/20231217.2/images/ubuntu/Ubuntu2204-Readme.md
2024-01-05T19:44:27.1787454Z Image Release: https://github.com/actions/runner-images/releases/tag/ubuntu22%2F20231217.2
2024-01-05T19:44:27.1789216Z ##[endgroup]
2024-01-05T19:44:27.1789979Z ##[group]Runner Image Provisioner
2024-01-05T19:44:27.1790665Z 2.0.321.1
2024-01-05T19:44:27.1791187Z ##[endgroup]
2024-01-05T19:44:27.1792839Z ##[group]GITHUB_TOKEN Permissions
2024-01-05T19:44:27.1795435Z Contents: write
2024-01-05T19:44:27.1796163Z Metadata: read
2024-01-05T19:44:27.1797091Z PullRequests: write
2024-01-05T19:44:27.1798000Z ##[endgroup]
2024-01-05T19:44:27.1802702Z Secret source: Actions
2024-01-05T19:44:27.1804849Z Prepare workflow directory
2024-01-05T19:44:27.2752083Z Prepare all required actions
2024-01-05T19:44:27.2998947Z Getting action download info
2024-01-05T19:44:27.5988441Z Download action repository 'google-github-actions/release-please-action@v4' (SHA:cc61a07e2da466bebbc19b3a7dd01d6aecb20d1e)
2024-01-05T19:44:28.6135610Z Complete job name: release-please
2024-01-05T19:44:28.7459935Z ##[group]Run google-github-actions/release-please-action@v4
2024-01-05T19:44:28.7460763Z with:
2024-01-05T19:44:28.7462191Z   token: ***
2024-01-05T19:44:28.7462729Z   release-type: terraform-module
2024-01-05T19:44:28.7463576Z   repo-url: org/repo
2024-01-05T19:44:28.7464463Z   github-api-url: https://api.github.com
2024-01-05T19:44:28.7465281Z   github-graphql-url: https://api.github.com/graphql
2024-01-05T19:44:28.7465918Z   fork: false
2024-01-05T19:44:28.7466530Z   include-component-in-tag: false
2024-01-05T19:44:28.7467087Z   skip-github-release: false
2024-01-05T19:44:28.7467567Z   skip-github-pull-request: false
2024-01-05T19:44:28.7468397Z ##[endgroup]
2024-01-05T19:44:29.3187398Z Running release-please version: 16.5.0
2024-01-05T19:44:29.8342193Z ✔ Looking for latest release on branch: main with prefix: 
2024-01-05T19:44:29.8344062Z ❯ Fetching merge commits on branch main with cursor: undefined
2024-01-05T19:44:31.4798072Z ❯ Found latest release pull request: 2 version: 0.1.0
2024-01-05T19:44:31.4808240Z ❯ Fetching releases with cursor undefined
2024-01-05T19:44:31.8471404Z ❯ found release for  Version {
2024-01-05T19:44:31.8472116Z   major: 0,
2024-01-05T19:44:31.8473363Z   minor: 1,
2024-01-05T19:44:31.8474004Z   patch: 0,
2024-01-05T19:44:31.8474467Z   preRelease: undefined,
2024-01-05T19:44:31.8475132Z   build: undefined
2024-01-05T19:44:31.8475592Z }
2024-01-05T19:44:31.8484100Z ❯ found 2 possible releases. [
2024-01-05T19:44:31.8484880Z   Version {
2024-01-05T19:44:31.8485405Z     major: 0,
2024-01-05T19:44:31.8486196Z     minor: 1,
2024-01-05T19:44:31.8486975Z     patch: 0,
2024-01-05T19:44:31.8487596Z     preRelease: undefined,
2024-01-05T19:44:31.8715523Z     build: undefined
2024-01-05T19:44:31.8716087Z   },
2024-01-05T19:44:31.8716654Z   Version {
2024-01-05T19:44:31.8716901Z     major: 0,
2024-01-05T19:44:31.8717311Z     minor: 1,
2024-01-05T19:44:31.8717557Z     patch: 0,
2024-01-05T19:44:31.8717863Z     preRelease: undefined,
2024-01-05T19:44:31.8718206Z     build: undefined
2024-01-05T19:44:31.8718473Z   }
2024-01-05T19:44:31.8718686Z ]
2024-01-05T19:44:31.8729306Z ✔ Building releases
2024-01-05T19:44:31.8729865Z ✔ Building strategies by path
2024-01-05T19:44:31.8730390Z ❯ .: terraform-module
2024-01-05T19:44:32.1565147Z ✔ Looking for latest release on branch: main with prefix: 
2024-01-05T19:44:32.1568769Z ❯ Fetching merge commits on branch main with cursor: undefined
2024-01-05T19:44:33.2547718Z ❯ Found latest release pull request: 2 version: 0.1.0
2024-01-05T19:44:33.2550071Z ❯ Fetching releases with cursor undefined
2024-01-05T19:44:33.6585992Z ❯ found release for  Version {
2024-01-05T19:44:33.6586539Z   major: 0,
2024-01-05T19:44:33.6586873Z   minor: 1,
2024-01-05T19:44:33.6587196Z   patch: 0,
2024-01-05T19:44:33.6587545Z   preRelease: undefined,
2024-01-05T19:44:33.6587934Z   build: undefined
2024-01-05T19:44:33.6588270Z }
2024-01-05T19:44:33.6592436Z ❯ found 2 possible releases. [
2024-01-05T19:44:33.6593002Z   Version {
2024-01-05T19:44:33.6593330Z     major: 0,
2024-01-05T19:44:33.6593970Z     minor: 1,
2024-01-05T19:44:33.6594292Z     patch: 0,
2024-01-05T19:44:33.6594620Z     preRelease: undefined,
2024-01-05T19:44:33.6595025Z     build: undefined
2024-01-05T19:44:33.6595376Z   },
2024-01-05T19:44:33.6595663Z   Version {
2024-01-05T19:44:33.6595958Z     major: 0,
2024-01-05T19:44:33.6596273Z     minor: 1,
2024-01-05T19:44:33.6596590Z     patch: 0,
2024-01-05T19:44:33.6596916Z     preRelease: undefined,
2024-01-05T19:44:33.6597340Z     build: undefined
2024-01-05T19:44:33.6597684Z   }
2024-01-05T19:44:33.6597963Z ]
2024-01-05T19:44:33.6611416Z ✔ Building pull requests
2024-01-05T19:44:33.6612254Z ✔ Building strategies by path
2024-01-05T19:44:33.6613197Z ❯ .: terraform-module
2024-01-05T19:44:33.6669080Z ✔ Collecting release commit SHAs
2024-01-05T19:44:33.6669858Z ❯ release search depth: 400
2024-01-05T19:44:33.6670580Z ❯ Fetching releases with cursor undefined
2024-01-05T19:44:34.0577353Z ❯ Found release for path ., v0.1.0
2024-01-05T19:44:34.0578449Z ❯ release for path: ., version: 0.1.0, sha: 61da1eb745fdff3bd6dd89eb4d89df4bd4c9fda1
2024-01-05T19:44:34.0579423Z ✔ Collecting commits since all latest releases
2024-01-05T19:44:34.0584035Z ❯ commit search depth: 500
2024-01-05T19:44:34.0585033Z ❯ Set(1) { '61da1eb745fdff3bd6dd89eb4d89df4bd4c9fda1' }
2024-01-05T19:44:34.0585946Z ❯ Fetching merge commits on branch main with cursor: undefined
2024-01-05T19:44:34.8898821Z ❯ Backfilling file list for commit: f03e93f9ca5a5cc02e25d276dbbe418fc5e2caa7
2024-01-05T19:44:35.1486248Z ❯ Found 1 files
2024-01-05T19:44:35.1498946Z ❯ Backfilling file list for commit: 51d0a0fcc08dfd6a9d84e6b0cb50c2adc983c892
2024-01-05T19:44:35.4143435Z ❯ Found 2 files
2024-01-05T19:44:35.4144197Z ❯ Backfilling file list for commit: 96d4875a4ca5eb83ce8e352295b761eb062889b1
2024-01-05T19:44:35.6645043Z ❯ Found 1 files
2024-01-05T19:44:35.6646154Z ❯ Backfilling file list for commit: 4a1f640fa7d3da8a4f1f067d9def824cd29ee645
2024-01-05T19:44:35.9858727Z ❯ Found 2 files
2024-01-05T19:44:35.9859795Z ❯ Backfilling file list for commit: 67b019668e6b14694d7a4162b8aacf48a79f3052
2024-01-05T19:44:36.2413454Z ❯ Found 2 files
2024-01-05T19:44:36.2415194Z ❯ Backfilling file list for commit: 0057667ed928e070f964f8ea2ee9930b88473b52
2024-01-05T19:44:36.4977527Z ❯ Found 1 files
2024-01-05T19:44:36.4978979Z ❯ Backfilling file list for commit: 44e4cac7c6654a11ea8d97b56f14fc792bc5f723
2024-01-05T19:44:36.7309190Z ❯ Found 2 files
2024-01-05T19:44:36.7310388Z ❯ Backfilling file list for commit: 16de3b9cdddbf5c6069d3f2d695f581e7359a5df
2024-01-05T19:44:36.9424641Z ❯ Found 1 files
2024-01-05T19:44:36.9425814Z ❯ Backfilling file list for commit: 216ead2a8d560956687c8e2f32c30d8aba98c60d
2024-01-05T19:44:37.1677303Z ❯ Found 1 files
2024-01-05T19:44:37.1678332Z ❯ Backfilling file list for commit: b8c7efc2b04bed3f2080614249a1d2442358a4a7
2024-01-05T19:44:37.3920904Z ❯ Found 1 files
2024-01-05T19:44:37.3922595Z ❯ Backfilling file list for commit: ab4955ff8756ff4cfa3036f6f122ccf88b2d47ca
2024-01-05T19:44:37.6092850Z ❯ Found 2 files
2024-01-05T19:44:37.6094774Z ❯ Backfilling file list for commit: 822d03781a3d2056625a9eeccf79b587b5c7042f
2024-01-05T19:44:37.8285313Z ❯ Found 1 files
2024-01-05T19:44:37.8286461Z ❯ Backfilling file list for commit: 3d94db62792fdc9fe8e10f1c6f6c73715795d43c
2024-01-05T19:44:38.0628303Z ❯ Found 2 files
2024-01-05T19:44:38.0629597Z ❯ Backfilling file list for commit: 7d359a2a31d02738816686e756032dffe2f8db49
2024-01-05T19:44:38.3147172Z ❯ Found 21 files
2024-01-05T19:44:38.3151681Z ❯ Backfilling file list for commit: 3df4d4c6707462980b9d8bbab1e8f401ae0d57e4
2024-01-05T19:44:38.5652078Z ❯ Found 2 files
2024-01-05T19:44:38.5653385Z ❯ Backfilling file list for commit: db9cd89cb465d70b7614e1d093fa4f87ef06a367
2024-01-05T19:44:38.7913844Z ❯ Found 5 files
2024-01-05T19:44:38.7915076Z ❯ Backfilling file list for commit: 9bdb86ff3a22154f7ca7adac0376e71714a3be12
2024-01-05T19:44:39.0088343Z ❯ Found 3 files
2024-01-05T19:44:39.0089524Z ❯ Backfilling file list for commit: 5f4eef0ce3e9842e538e61b01869a464b46f8d74
2024-01-05T19:44:39.2244382Z ❯ Found 1 files
2024-01-05T19:44:39.2245999Z ❯ Backfilling file list for commit: 69c071b18bbe6a6aba86b374df81c4ec09157313
2024-01-05T19:44:39.4732936Z ❯ Found 1 files
2024-01-05T19:44:39.4734182Z ❯ Backfilling file list for commit: 15f12e054fe776f6b0f84b8ebbafd13351219736
2024-01-05T19:44:39.7097013Z ❯ Found 1 files
2024-01-05T19:44:39.7097795Z ❯ Backfilling file list for commit: 19f2ffcb593814a2b96cebdecb018c17d9ff6691
2024-01-05T19:44:39.9560423Z ❯ Found 3 files
2024-01-05T19:44:39.9561844Z ❯ Backfilling file list for commit: 06fd82183d52614d5e3a0f1feaee5e7f898468fa
2024-01-05T19:44:40.1951657Z ❯ Found 2 files
2024-01-05T19:44:40.1954757Z ❯ Backfilling file list for commit: 3499d6400812632c30937cce268e68e7c87f49e7
2024-01-05T19:44:40.4161947Z ❯ Found 3 files
2024-01-05T19:44:40.4163392Z ✔ Splitting 7 commits by path
2024-01-05T19:44:40.4176300Z ✔ Building candidate release pull request for path: .
2024-01-05T19:44:40.4177574Z ❯ type: terraform-module
2024-01-05T19:44:40.4178032Z ❯ targetBranch: main
2024-01-05T19:44:40.4220373Z ❯ commits: 7
2024-01-05T19:44:40.4223706Z ✔ Considering: 7 commits
2024-01-05T19:44:40.4228120Z ❯ component: 
2024-01-05T19:44:40.4228531Z ❯ pull request title pattern: undefined
2024-01-05T19:44:40.4598126Z ✔ No user facing commits found since 61da1eb745fdff3bd6dd89eb4d89df4bd4c9fda1 - skipping
2024-01-05T19:44:40.5234815Z Cleaning up orphan processes
```

**Comment by atsu85** [comment]
i've stumbled upon the same bug with `type: simple` and `release-type: node` so probably release strategy doesn't matter.

@kstevensonnv, perhaps you'll rename the bug title from
> PR not created on prefix 'deps' using 'terraform-module' release-strategy

to perhaps smth like
> PR not created on prefix 'deps'

or 
> PR not created on prefix 'deps' unlike documented

?
